<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginPage</class>
 <widget class="QWidget" name="loginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="errorBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>171</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
